populate/src/main/resources/seed/portals/ourhealth/studies/ourheart/enrollees/seed.json

```diff
@@ -0,0 +1,68 @@
6ohQIAAAQIEGhAQsBpooiEQIECAAAECsQQErFj9UA0BAgQIECDQgICA1UATDYEAAQIECBCIJSBgxeqHaggQIECAAIEGBASsBppoCAQIECBAgEAsAQErVj9UQ4AAAQIECDQgIGA10ERDIECAAAECBGIJCFix+qEaAgQIECBAoAEBAauBJhoCAQIECBAgEEtAwIrVD9UQIECAAAECDQgIWA000RAIECBAgACBWAICVqx+qIYAAQIECBBoQEDAaqCJhkCAAAECBAjEEhCwYvVDNQQIECBAgEADAgJWA000BAIECBAgQCCWgIAVqx+qIUCAAAECBBoQELAaaKIhECBAgAABArEEBKxY/VANAQIECBAg0ICAgNVAEw2BAAECBAgQiCUgYMXqh2oIECBAgACBBgQErAaaaAgECBAgQIBALAEBK1Y/VEOAAAECBAg0ICBgNdBEQyBAgAABAgRiCQhYsfqhGgIECBAgQKABAQGrgSYaAgECBAgQIBBLQMCK1Q/VECBAgAABAg0ICFgNNNEQCBAgQIAAgVgCAlasfqiGAAECBAgQaEBAwGqgiYZAgAABAgQIxBIQsGL1QzUECBAgQIBAAwICVgNNNAQCBAgQIEAgloCAFasfqiFAgAABAgQaEBCwGmiiIRAgQIAAAQKxBASsWP1QDQECBAgQINCAgIDVQBMNgQABAgQIEIglIGDF6odqCBAgQIAAgQYEBKwGmmgIBAgQIECAQCwBAStWP1RDgAABAgQINCAgYDXQREMgQIAAAQIEYgkIWLH6oRoCBAgQIECgAQEBq4EmGgIBAgQIECAQS0DAitUP1RAgQIAAAQINCAhYDTTREAgQIECAAIFYAgJWrH6ohgABAgQIEGhAQMBqoImGQIAAAQIECMQSELBi9UM1BAgQIECAQAMCAlYDTTQEAgQIECBAIJaAgBWrH6ohQICSBr/0AAAF5klEQVQAAQIEGhAQsBpooiEQIECAAAECsQQErFj9UA0BAgQIECDQgICA1UATDYEAAQIECBCIJSBgxeqHaggQIECAAIEGBASsBppoCAQIECBAgEAsAQErVj9UQ4AAAQIECDQgIGA10ERDIECAAAECBGIJCFix+qEaAgQIECBAoAEBAauBJhoCAQIECBAgEEtAwIrVD9UQIECAAAECDQgIWA000RAIECBAgACBWAICVqx+qIYAAQIECBBoQEDAaqCJhkCAAAECBAjEEhCwYvVDNQQIECBAgEADAgJWA000BAIECBAgQCCWgIAVqx+qIUCAAAECBBoQELAaaKIhECBAgAABArEEBKxY/VANAQIECBAg0ICAgNVAEw2BAAECBAgQiCUgYMXqh2oIECBAgACBBgQErAaaaAgECBAgQIBALAEBK1Y/VEOAAAECBAg0ICBgNdBEQyBAgAABAgRiCQhYsfqhGgIECBAgQKABAQGrgSYaAgECBAgQIBBLQMCK1Q/VECBAgAABAg0ICFgNNNEQCBAgQIAAgVgCAlasfqiGAAECBAgQaEBAwGqgiYZAgAABAgQIxBIQsGL1QzUECBAgQIBAAwICVgNNNAQCBAgQIEAgloCAFasfqiFAgAABAgQaEBCwGmiiIRAgQIAAAQKxBASsWP1QDQECBAgQINCAgIDVQBMNgQABAgQIEIglIGDF6odqCBAgQIAAgQYEBKwGmmgIBAgQIECAQCwBAStWP1RDgAABAgQINCAgYDXQREMgQIAAAQIEYgkIWLH6oRoCBAgQIECgAQEBq4EmGgIBAgQIECAQS0DAitUP1RAgQIAAAQINCAhYDTTREAgQIECAAIFYAgJWrH6ohgABAgQIEGhAQMBqoImGQIAAAQIECMQSELBi9UM1BAgQIECAQAMCAlYDTTQEAgQIECBAIJaAgBWrH6ohQIAAAQIEGhAQsBpooiEQIECAAAECsQQErFj9UA0BAgQIECDQgICA1UATDYEAAQIECBCIJSBgxeqHaggQIECAAIEGBASsBppoCAQIECBAgEAsAQErVj9UQ4AAAQIECDQgIGA10ERDIECAAAECBGIJCFix+qEaAgQIECBAoAEBAauBJhoCAQIECBAgEEtAwIrVD9UQIECAAAECDQgIWA000RAIECBAgACBWAICVqx+qIYAAQIECBBoQEDAaqCJhkCAAAECBAjEEhCwYvVDNQQIECBAgEADAgJWA000BAIECBAgQCCWgIAVqx+qIUCAAAECBBoQELAaaKIhECBAgAABArEEBKxY/VANAQIECBAg0ICAgNVAEw2BAAECBAgQiCUgYMXqh2oIECBAgACBBgQErAaaaAgECBAgQIBALAEBK1Y/VEOAAAECBAg0ICBgNdBEQyBAgAABAgRiCQhYsfqhGgIECBAgQKABAQGrgSYaAgECBAgQIBBLQMCK1Q/VECBAgAABAg0ICFgNNNEQCBAgQIAAgVgCAlasfqiGAAECBAgQaEBAwGqgiYZAgAABAgQIxBIQsGL1QzUECBAgQIBAAwICVgNNNAQCBAgQIEAgloCAFasfqiFAgAABAgQaEBCwGmiiIRAgQIAAAQKxBASsWP1QDQECBAgQINCAgIDVQBMNgQABAgQIEIglIGDF6odqCBAgQIAAgQYEBKwGmmgIBAgQIECAQCwBAStWP1RDgAABAgQINCAgYDXQREMgQIAAAQIEYgkIWLH6oRoCBAgQIECgAQEBq4EmGgIBAgQIECAQS0DAitUP1RAgQIAAAQINCAhYDTTREAgQIECAAIFYAgJWrH6ohgABAgQIEGhAQMBqoImGQIAAAQIECMQSELBi9UM1BAgQIECAQAMCAlYDTTQEAgQIECBAIJaAgBWrH6ohQIAAAQIEGhAQsBpooiEQIECAAAECsQQErFj9UA0BAgQIECDQgICA1UATDYEAAQIECBCIJSBgxeqHaggQIECAAIEGBASsBppoCAQIECBAgEAsAQErVj9UQ4AAAQIECDQg8P/3ZrhBnabaAQAAAABJRU5ErkJggg=="
```

Comment:
to save space and readability (and possibly reduce insertion time), you could replace this with a smaller image (e.g. a base64 encoding of a small icon or something)

Reply:
Reduced by ~90% in c6bad97a26f9d0068f8898777a64022ec0c9c28d